### ISSUE TITLE
Crash on start when using Regex as a normal global property

### ISSUE THREAD

**Issue by soywiz** [JetBrains/kotlin-native#1331]
https://github.com/korlibs/klock/pull/3/files#diff-860e7b4b097dd9574c2c194b771bbad4R554

```kotlin
// @TODO: Kotlin Native crash on start
//private val formatRegex = Regex("%([-]?\\d+)?(\\w)")

// Any of these two options work fine:
private val formatRegex get() = Regex("%([-]?\\d+)?(\\w)")
private val formatRegex by lazy { Regex("%([-]?\\d+)?(\\w)") }
```

It doesn't happen if I change that property to be a getter or a lazy property.

**Comment by olonho** [comment]
Yes, it is an init order problem. We're working on appropriate fix for list problem, so that dependent libs ctors are initalized first.

**Comment by olonho** [comment]
Fixed with 346e1aefcf2424eecba4722741d112211254ee29
